src/javascripts/helpers/data/chaosData.js

```diff
@@ -0,0 +1,52 @@
+// import equipmentData from './equipmentData';
+import axios from 'axios';
+
+const randomEquipment = (category) => new Promise((resolve, reject) => {
+  axios
+    .get(`https://nutshell-part-two.firebaseio.com/${category}.json`)
+    .then((response) => {
+      const arrayLength = Object.values(response.data).length;
+      const randomNumber = Math.floor(Math.random() * arrayLength);
+      const choatic = Object.values(response.data)[randomNumber];
+      resolve(choatic.name);
+    })
+    .catch((error) => reject(error));
+
+  // equipmentData
+  //   .getEquipment()
+  //   .then((response) => {
+  //     const arrayLength = response.length;
+  //     const randomNumber = Math.floor(Math.random() * arrayLength);
+  //     const brokenEquipment = response[randomNumber];
+  //     resolve(brokenEquipment);
+  //   })
+  //   .catch((error) => reject(error));
+});
+
+const chaosMonkey = () => new Promise((resolve, reject) => {
+  const chaosArray = ['staff', 'equipment', 'rides'];
+  const category = chaosArray[Math.floor(Math.random() * 3)];
+
+  // TODO: Update the string/resolve based on the selected category
+  randomEquipment(category)
+    .then((response) => {
+      let selectedCat = `broken the ${response}`;
+      if (category === 'staff') {
+        selectedCat = `kidnapped ${response}`;
+      }
+      //   selectedCat = ...
```

Comment:
remove

Reply:
Is this the updated code?

Reply:
We've removed this comment as well

Reply:
This is the updated code, since the rides and equipment both break, I have the default selectedCat set equal to``broken the ${response}`` and selectedCat only changes if the category is "staff" to ``kidnapped ${response}``.